packages/RatPack.package/RPModel.class/class/updateObject.withJson..st

```diff
@@ -0,0 +1,11 @@
+object conversion
+updateObject: aRPModelClassObject withJson: aJsonString
+
+	| newObject aJsonObject|
+	
+	aJsonObject := self createJsonObject: aJsonString. 
+	newObject := aRPModelClassObject. 
```

Comment:
Why do we need this assignment?

Reply:
I'd say this is called `updateObject`, so I'd expect this to update the object, not create an updated copy. Personally, I don't like making copies.

Reply:
I do not understand why you want the copy in the first place?

Reply:
I don’t like copies neither but I don’t know why also don’t like the class methods modifying.

But it’s inelegant anyway so I’ll be fine with anything.